src/precompile/modexp.rs

```diff
@@ -1,20 +1,21 @@
 use revm::{
     precompile::{
-        modexp::{self, berlin_gas_calc, run_inner},
+        modexp::{berlin_gas_calc, run_inner},
+        u64_to_address,
         utilities::right_pad_with_offset,
-        PrecompileError, PrecompileResult, PrecompileWithAddress,
+        Precompile, PrecompileError, PrecompileId, PrecompileResult,
     },
     primitives::{Address, U256},
 };
 
 /// The MODEXP precompile address.
-pub const ADDRESS: Address = modexp::BYZANTIUM.0;
```

Comment:
This used to work when `modexp::BYZANTIUM` was a `PrecompileWithAddress` (which is a tuple) but doesn't with `Precompile` which is a struct. I'm not sure why, it seems to be a compiler limitation.